### PR TITLE
KOGITO-1181: [DMN Designer] Default Output value for Decision Table are not persisted

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.redhat.com/browse/KOGITO-1181

### CODE REVIEW

**Comment by manstis** [comment]
@jomarko It is **impossible** to write unit tests for this which is a worry.

We could consider adding back my [hack](https://github.com/manstis/kie-wb-common/commit/02661f1dc87adf173562069617ba5fad0a77f834#diff-a1d7ded22134e0bbb46572d6cd87d58cR119) for _unit-test_ type tests.. then it could be tested.

**Comment by manstis** [comment]
To test; either:-

**_Using the WAR_**
- In `kie-wb-common-dmn-webapp-kogito-runtime`
- `mvn clean install -DskipTests`
- Unzip WAR and open `index.html` in a Chrome
- With DevTools, Console enter `gwtEditorBeans.get("DMNDiagramEditor").get().setContent("", "xxx")` where `xxx` is the content of the "single line" file attached to the JIRA

**_Using VSCode extension_**
- In `kie-wb-common-dmn-webapp-kogito-runtime`
- `mvn clean install -DskipTests`
- [Change](https://github.com/kiegroup/kogito-tooling/blob/master/packages/kie-bc-editors-unpacked/pom.xml#L40) `kogito-tooling` to use local `-SNAPSHOT`
- Build `kogito-tooling`
- Install VSIX file in VSCode
- Open file attached to JIRA

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-common/src/main/java/org/kie/workbench/common/dmn/webapp/kogito/common/client/converters/model/OutputClauseLiteralExpressionPropertyConverter.java [hunk 1]
```diff
@@ -47,4 +54,19 @@ public static OutputClauseLiteralExpression wbFromDMN(final JSITLiteralExpressio
...
+
+    /**
+     * Returns a JSITLiteralExpression of null if the text of the OutputClauseLiteralExpression is empty.
+     * @param wb
+     * @return
+     */
+    public static JSITLiteralExpression dmnFromWB(final OutputClauseLiteralExpression wb) {
+        if (Objects.isNull(wb)) {
+            return null;
+        } else if (StringUtils.isEmpty(wb.getText().getValue())) {
```
Comment (danielzhe) [inline kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-common/src/main/java/org/kie/workbench/common/dmn/webapp/kogito/common/client/converters/model/OutputClauseLiteralExpressionPropertyConverter.java]: `wb.getText()` never returns `null`?

**Comment by danielzhe** [comment]
> 
> 
> To test; either:-
> 
> **_Using the WAR_**
> 
>     * In `kie-wb-common-dmn-webapp-kogito-runtime`
> 
>     * `mvn clean install -DskipTests`
> 
>     * Unzip WAR and open `index.html` in a Chrome
> 
>     * With DevTools, Console enter `gwtEditorBeans.get("DMNDiagramEditor").get().setContent("", "xxx")` where `xxx` is the content of the "single line" file attached to the JIRA
> 
> 
> **_Using VSCode extension_**
> 
>     * In `kie-wb-common-dmn-webapp-kogito-runtime`
> 
>     * `mvn clean install -DskipTests`
> 
>     * [Change](https://github.com/kiegroup/kogito-tooling/blob/master/packages/kie-bc-editors-unpacked/pom.xml#L40) `kogito-tooling` to use local `-SNAPSHOT`
> 
>     * Build `kogito-tooling`
> 
>     * Install VSIX file in VSCode
> 
>     * Open file attached to JIRA

I'm saving this comment because I know it will be very useful in the future. 
Thank you!

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-common/src/main/java/org/kie/workbench/common/dmn/webapp/kogito/common/client/converters/model/OutputClauseLiteralExpressionPropertyConverter.java (hunk 1 above)
Comment (manstis) [reply kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-common/src/main/java/org/kie/workbench/common/dmn/webapp/kogito/common/client/converters/model/OutputClauseLiteralExpressionPropertyConverter.java]: Hi @danielzhe there is a `public` _setter_ for `Text` that does not do a `null` check. Therefore it's possible that it _could_ be `null` although that scenario is probably a bug in it's own right! However I've added a new commit checking that `getText()` is non-null now too. Thank-you .

**Comment by manstis** [comment]
Hi @jomarko this PR is ready for your, or @dupliaka, review now please.

**Comment by jomarko** [comment]
I started long time ago https://github.com/kiegroup/kie-wb-common/pull/3171

**Pull request merged**